data/favicon.json

```diff
@@ -324,6 +327,7 @@
 	"videolan.org": "videolan.org.ico",
 	"vistaprint.it": "vistaprint.it.png",
 	"voelkner.de": "voelkner.de.ico",
+	"watchlistmailings.de": "ing.de.png",
```

Comment:
What is the relationship between `watchlistmailings.de` and `ing.de`?
Could not find any with a quick search, so I don't think maisl signed by `watchlistmailings.de` shoudl show the favicon of `ing.de`.

Reply:
Ing is using watchlistmailings.de as mail domain for it's "Watchlist limit alerts":
https://www.ing.de/wertpapiere/watchlist-funktionen/